src/ai/backend/manager/models/vfolder.py

```diff
@@ -1020,13 +1020,19 @@ class Meta:
     def from_row(cls, ctx: GraphQueryContext, row: Row) -> Optional[VirtualFolder]:
         if row is None:
             return None
+
+        try:
+            user_email = row["users_email"]
+        except BaseException:
+            user_email = None
+
         return cls(
             id=row["id"],
             host=row["host"],
             quota_scope_id=row["quota_scope_id"],
             name=row["name"],
             user=row["user"],
-            user_email=row["users_email"],
+            user_email=user_email,
```

Comment:
```suggestion
            user_email= row["users_email"] if "user_email" in row else None
```
Could you try this? Please remove the `try-catch` block if this works.

Reply:
I was approaching in the wrong way...😅
(ex: `row["users_email"] if row["user_email"] else None`)

And it works very well!